cmd/migrate_storage.go

```diff
@@ -25,13 +27,13 @@ import (
 var CmdMigrateStorage = cli.Command{
 	Name:        "migrate-storage",
 	Usage:       "Migrate the storage",
-	Description: "This is a command for migrating storage.",
+	Description: "This is a command for migrating Gitea configured storage to a parameter indicated storage.",
 	Action:      runMigrateStorage,
 	Flags: []cli.Flag{
 		cli.StringFlag{
 			Name:  "type, t",
 			Value: "",
-			Usage: "Kinds of files to migrate, currently only 'attachments' is supported",
+			Usage: "Kinds of files to migrate, could be one of 'attachments', 'lfs', 'avatars', 'repo-avatars', 'repo-archivers', 'packages'",
```

Comment:
The parameter is called `type` -> that suggests that the usage should use `Type` too. (I know this is the opposite of what I have argued above...)

Further, only one `type` of storage is allowed so a singular is required:

```suggestion
			Usage: "Type of stored files to copy.  Allowed types: 'attachments', 'lfs', 'avatars', 'repo-avatars', 'repo-archivers', 'packages'",
```

I am concerned thought that `--type` isn't a great name for this parameter but I can't immediately think of a better name.